public_dropin_environments/triton_server/start_server.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Copyright 2024 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+echo "Starting Custom Model environment with Triton inference server"
+
+if [ "${ENABLE_CUSTOM_MODEL_RUNTIME_ENV_DUMP}" = 1 ]; then
+    echo "Environment variables:"
+    env
+fi
+
+export MODEL_DIR="${CODE_DIR}/model_repository/"
+
+echo
+echo "Executing command: tritonserver --model-repository=${MODEL_DIR}"
+echo
+exec nohup tritonserver --model-repository=${MODEL_DIR} > log.txt 2>&1 &
```

Comment:
```suggestion
nohup tritonserver --model-repository=${MODEL_DIR} > log.txt 2>&1 &
```
I don't think `exec` is needed here -- I'm a little surprised it works TBH. Can you explain why you need it?

Also, I'm worried about sending all output to the `log.txt`. From my experience playing around with this stuff, if there is a problem with the model file, the output will be from Trition explaining what is wrong and with this line, I don't see how the user will ever see that? What is the problem with just leaving the output going to stdout/stderr -- does this result in the output getting intermixed, is it still readable?

Reply:
The `exec` has no effect here, the `nohup &` is sufficient to run Triton in the background process. Removed the `exec` command.

Reply:
> What is the problem with just leaving the output going to stdout/stderr -- does this result in the output getting intermixed, is it still readable

I just wanted to see output from Flask proxy more, than output from Triton. But for user's containers, I agree Triton logs are more important. I'll remove output redirection